bedrock/firefox/templates/firefox/new/scene1.html

```diff
@@ -141,9 +141,13 @@ <h2 class="good">{{_('Freedom is yours') }}</h2>
       <section class="section-current-platform">
         {# L10n: span's below are for visual formatting and line break. #}
         <h4 class="heading-windows">{{_('Download Firefox <span>for Windows</span>') }}</h4>
-        <h4 class="heading-osx">{{_('Download Firefox <span>for OS X</span>') }}</h4>
-        <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
-
+        {% if l10n_has_tag('macos_branding') %}
+          <h4 class="heading-osx">{{_('Download Firefox <span>for macOS</span>') }}</h4>
+        {% else %}
+          <h4 class="heading-osx">{{_('Download Firefox <span>for OS X</span>') }}</h4>
+        {% endif %}
+          <h4 class="heading-linux">{{_('Download Firefox <span>for Linux</span>') }}</h4>
```

Comment:
Also note that indentation went off here (it would be easy to spot when you remove the if structure).